framework/src/util/Properties.cpp

```diff
@@ -24,100 +24,105 @@
 
 #include <limits>
 #include <stdexcept>
-#ifdef US_PLATFORM_WINDOWS
-#  include <string.h>
-#  define ci_compare strnicmp
-#else
-#  include <strings.h>
-#  define ci_compare strncasecmp
-#endif
+
+#include "PropsCheck.h"
 
 US_MSVC_PUSH_DISABLE_WARNING(4996)
 
 namespace cppmicroservices {
 
 const Any Properties::emptyAny;
 
+Properties::Properties(AnyMap& p)
```

Comment:
Should this be a const&? Or perhaps a &&? A reference implies that the values can be changed, which should not happen with the way it's coded, right?

Reply:
I now see the other versions of the constructor below... I think this constructor should probably be removed.

Reply:
Fixed.